HaskellDates.hsproj/HaskellDates.hs

```diff
@@ -0,0 +1,62 @@
+import Data.Time
+import Data.Time.Calendar.OrdinalDate
+import Control.Applicative
+
+data Event = Event {
+  date :: Day,
+  daysOfWeek :: [Int],
+  week :: Int
+}
+
+-- WeekDay for debugging purposes
+newtype WeekDay = WeekDay Int
+
+instance Show WeekDay where
+  show (WeekDay x)
+    | x == 0 = "Sunday"
+    | x == 1 = "Monday"
+    | x == 2 = "Tuesday"
+    | x == 3 = "Wednesday"
+    | x == 4 = "Thursday"
+    | x == 5 = "Friday"
+    | x == 6 = "Saturday"
```

Comment:
If you want, you can pattern match on numeric literals:

```hs
show (WeekDay 0) = "Sunday"
show (WeekDay 1) = "Monday"
```

Also, you should make this exhaustive -- even if you `error` on unexpected values, it's better to be explicit about it.